api/repositories/process_repository_test.go

```diff
@@ -138,87 +149,56 @@ var _ = Describe("ProcessRepo", func() {
-				Expect(k8sClient.Create(context.Background(), cfApp1)).To(Succeed())
-
-				cfApp2 = initializeAppCR("test-app2", app2GUID, namespace2.Name)
-				Expect(k8sClient.Create(context.Background(), cfApp2)).To(Succeed())
-
-				cfProcess1 = initializeProcessCR(process1GUID, namespace1.Name, app1GUID)
-				Expect(k8sClient.Create(context.Background(), cfProcess1)).To(Succeed())
-
-				cfProcess2 = initializeProcessCR(process1GUID, namespace2.Name, app2GUID)
-				Expect(k8sClient.Create(context.Background(), cfProcess2)).To(Succeed())
+				duplicateProcess := initializeProcessCR(process1GUID, space2.Name, app2GUID)
+				Expect(k8sClient.Create(context.Background(), duplicateProcess)).To(Succeed())
 			})
 
-			//AfterEach(func() {
-			//	k8sClient.Delete(context.Background(), cfApp1)
-			//	k8sClient.Delete(context.Background(), cfApp2)
-			//	k8sClient.Delete(context.Background(), cfProcess1)
-			//	k8sClient.Delete(context.Background(), cfProcess2)
-			//})
-
 			It("returns an error", func() {
-				_, err := processRepo.GetProcess(testCtx, authInfo, process1GUID)
-				Expect(err).To(HaveOccurred())
-				Expect(err).To(MatchError("duplicate processes exist"))
+				Expect(getErr).To(MatchError("duplicate processes exist"))
 			})
 		})
 
 		When("no Processes exist", func() {
+			BeforeEach(func() {
+				process1GUID = "does-not-exist"
+			})
+
 			It("returns an error", func() {
-				_, err := processRepo.GetProcess(testCtx, authInfo, "i don't exist")
-				Expect(err).To(HaveOccurred())
-				Expect(err).To(MatchError(NotFoundError{ResourceType: "Process"}))
+				Expect(getErr).To(MatchError(NotFoundError{ResourceType: "Process"}))
 			})
 		})
 	})
 
 	Describe("ListProcesses", func() {
-		var (
-			cfApp1 *workloadsv1alpha1.CFApp
-			cfApp2 *workloadsv1alpha1.CFApp
-
-			cfProcess1 *workloadsv1alpha1.CFProcess
-			cfProcess2 *workloadsv1alpha1.CFProcess
-		)
-
 		BeforeEach(func() {
-			cfApp1 = initializeAppCR("test-app1", app1GUID, namespace.Name)
-			Expect(k8sClient.Create(testCtx, cfApp1)).To(Succeed())
-
-			cfApp2 = initializeAppCR("test-app2", app2GUID, namespace.Name)
-			Expect(k8sClient.Create(testCtx, cfApp2)).To(Succeed())
+			cfApp1 = initializeAppCR("test-app1", app1GUID, space1.Name)
+			Expect(k8sClient.Create(context.Background(), cfApp1)).To(Succeed())
 
-			cfProcess1 = initializeProcessCR(process1GUID, namespace.Name, app1GUID)
-			Expect(k8sClient.Create(testCtx, cfProcess1)).To(Succeed())
+			cfProcess1 = initializeProcessCR(process1GUID, space1.Name, app1GUID)
+			Expect(k8sClient.Create(context.Background(), cfProcess1)).To(Succeed())
 
-			cfProcess2 = initializeProcessCR(process2GUID, namespace.Name, app1GUID)
-			Expect(k8sClient.Create(testCtx, cfProcess2)).To(Succeed())
+			cfProcess2 = initializeProcessCR(process2GUID, space1.Name, app1GUID)
+			Expect(k8sClient.Create(context.Background(), cfProcess2)).To(Succeed())
 		})
 
 		When("on the happy path", func() {
 			When("spaceGUID is not empty", func() {
```

Comment:
I believe that this is the default case, it should no be inside a dedicated context